bundles/pascal/README.md

```diff
@@ -0,0 +1,12 @@
+# Nim bundle
```

Comment:
@shalabhc I have *no* clue how the hell I managed to miss something that obvious...but...fixed!